mlflow/models/model.py

```diff
@@ -564,7 +582,8 @@ def load(cls, path):
         if os.path.isdir(path):
             path = os.path.join(path, MLMODEL_FILE_NAME)
         with open(path) as f:
-            return cls.from_dict(yaml.safe_load(f.read()))
+            model_cfg = yaml.safe_load(f.read())
+            return cls.from_dict(recursive_fun_dict(model_cfg, convert_path))
```

Comment:
uh ? Can we generate the correct path when saving model ?

Reply:
It is possible. But anyway it will be necessary to reload it with the correct format when containerizing mlflow in different OS images. E.g. training model locally on Windows and deploying a debian-based image. So I guess it's better to keep the posix path notation when saving the model.